test/parallel/test-dgram-send-callback-buffer-length.js

```diff
@@ -11,6 +11,7 @@ const offset = 20;
 const len = buf.length - offset;
 
 const messageSent = common.mustCall(function messageSent(err, bytes) {
+  assert.ifError(err);
   assert.notStrictEqual(bytes, buf.length);
```

Comment:
Nit: not related to this PR but I think this assertion is redundant. We can keep only the one on the next line.